config.go

```diff
@@ -30,11 +30,24 @@ type WAFConfig interface {
 	// WithContentInjection enables content injection.
 	WithContentInjection() WAFConfig
 
-	// WithRequestBodyAccess configures access to the request body.
-	WithRequestBodyAccess(config RequestBodyConfig) WAFConfig
+	// WithRequestBodyAccess enables access to the request body.
+	WithRequestBodyAccess() WAFConfig
 
-	// WithResponseBodyAccess configures access to the response body.
-	WithResponseBodyAccess(config ResponseBodyConfig) WAFConfig
+	// WithRequestBodyLimit sets the maximum number of bytes that can be read from the request body. Bytes beyond that set
+	// in WithInMemoryLimit will be buffered to disk.
+	WithRequestBodyBytesLimit(limit int64) WAFConfig
```

Comment:
Sorry didn't comment yet since I think it's the same as what @jptosso was saying but let's use `int`, not `int64`. They are the same size on any practical system but anything other than `int` almost always results in annoying conversion for users quite often